backend/dataall/db/api/environment.py

```diff
@@ -1087,98 +899,6 @@ def paginated_environment_networks(
             page_size=data.get('pageSize', 10),
         ).to_dict()
 
-    @staticmethod
-    @has_resource_perm(permissions.LIST_ENVIRONMENT_DATASETS)
-    def paginated_environment_data_items(
```

Comment:
just to double-check: this function was not used at all right? (and therefore not moved to the new dataset share service)

Reply:
It was moved to `DatasetShareService`